homeassistant/components/hunterdouglas_powerview/const.py

```diff
@@ -81,5 +89,10 @@
     FIRMWARE_NAME: LEGACY_DEVICE_MODEL,
 }
 
-
 API_PATH_FWVERSION = "api/fwversion"
+
+POSKIND_NONE = 0
+POSKIND_PRIMARY = 1
+POSKIND_SECONDARY = 2
+POSKIND_VANE = 3
+POSKIND_ERROR = 4
```

Comment:
That makes sense if we were importing them from the api module (which would be preferred), but since they are created in Home Assistant, we need to follow Home Assistant's standards here for naming